src/components/CallToAction.jsx

```diff
@@ -19,11 +19,16 @@ export function CallToAction() {
         <div className='mx-auto max-w-lg text-center'>
           <h2 className='font-display text-3xl tracking-tight text-white sm:text-4xl'>Join our conversation</h2>
           <p className='mt-4 text-lg tracking-tight text-white'>
-            Before we officially launch the site, feel free to join us on Discord.
+            Before we officially launch the site, feel free to join us on Discord and Twitch.
           </p>
-          <Button href='https://discord.com/invite/nNtVfKddDD' target='_blank' color='white' className='mt-10'>
-            <span className='text-blue-600'>Join us on Discord</span>
-          </Button>
+          <div className='mt-10 flex justify-center gap-x-6'>
+            <Button href='https://discord.com/invite/nNtVfKddDD' target='_blank' color='white'>
+              <span className='text-blue-600'>Join us on Discord</span>
+            </Button>
+            <Button href='https://www.twitch.tv/ykdojo' target='_blank' color='white'>
+              <span className='text-violet-800'>Join us on Twitch</span>
```

Comment:
I tried different colors here. Purple 600 seems to work well: https://tailwindcss.com/docs/customizing-colors